js/app/modules/layout/topMenu.js

```diff
@@ -82,7 +91,7 @@ var TopMenu = new Module.Class({
         let content = this.create_submodule('content');
         this._grid.add(content);
 
-        if (content instanceof Gtk.ScrolledWindow) {
+        if (this.hide_on_scroll && content instanceof Gtk.ScrolledWindow) {
```

Comment:
I don't think this check is correct, it's cutting out other logic if `hide_on_scroll` is turned off.